src/app/handlers/blacklist.handler.ts

```diff
@@ -2,37 +2,56 @@ import { TextChannel } from 'discord.js';
 import Constants from '../../common/constants';
 import { IContainer, IHandler, IMessage, ClassType } from '../../common/types';
 import { Moderation } from '../../services/moderation.service';
+import { readFileSync } from 'fs';
+
+const vader = require('vader-sentiment');
+const blockedWordsLine = (readFileSync('src/common/wordblacklist.txt', 'utf-8')).split(/\r?\n/);
```

Comment:
rename to 'blacklistedWords'

Reply:
also make sure that if the file doesn't exist we fail gracefully and have an empty list of words.